src/common/components/elements/ListingPreview.tsx

```diff
@@ -275,20 +301,28 @@ export function ListingPreview(listing: Listing) {
   }, [nftMetadata?.uri]);
 
   // shows up to 2 decimals, but removes pointless 0s
-  const displayPrice = Number((getListingPrice(listing) * 0.000000001).toFixed(2));
+  const displayPrice = getFormatedListingPrice(listing);
 
   // no subdomain means it's a shell/skeleton
   if (loading || !listing.subdomain) {
     return <SkeletonListing />;
   }
 
   return (
-    <div ref={cardRef}>
+    <div
+      ref={cardRef}
+      onClick={() => {
+        track('Select listing', {
```

Comment:
The referrer not enough? Click throughs out of the site into a storefront are already tracked by the storefronts tracking as a referrer.

Reply:
I think using only the referrer leaves potentially valuable data on the table, and I think it will also be easier to build reports on this event, put it into funnels etc.

I really want to keep it